src/libsync/account.cpp

```diff
@@ -488,7 +488,29 @@ void Account::slotHandleSslErrors(QNetworkReply *reply, QList<QSslError> errors)
 
 void Account::slotCredentialsFetched()
 {
-    emit credentialsFetched(_credentials.data());
+    if (_davUser.isEmpty()) {
+        qCDebug(lcAccount) << "User id not set. Fetch it.";
+        const auto fetchUserNameJob = new JsonApiJob(sharedFromThis(), QStringLiteral("/ocs/v1.php/cloud/user"));
+        connect(fetchUserNameJob, &JsonApiJob::jsonReceived, this, [this, fetchUserNameJob](const QJsonDocument &json, int statusCode) {
+            fetchUserNameJob->deleteLater();
+            if (statusCode != 100) {
```

Comment:
so error is managed there and we always continue ?

Reply:
Ah no I revert my last comment. That will break the logic which forces a fetch of the user id. So I think it's best to continue and just log the error.